src/app/components/core/asset-readings/readings-graph/readings-graph.component.ts

```diff
@@ -57,8 +57,9 @@ export class ReadingsGraphComponent implements OnDestroy {
   public pauseTime: number = Date.now();
   public backwardReadingCounter: number = 0;
   public imageReadingsDimensions = {width: 0, height: 0, depth: 0};
-  public readingTimestamps = {start : "", end: ""};
+  public infoTextTimestamps = {start : "", end: ""};
   public graphStartTimestamp: Date;
+  public zoomObject = {minZoomValue: 1, isZoomed: false};
```

Comment:
don't use type like list/object/int etc. as suffix or prefix.  zoomConfig is the actual name;